presto-main/src/main/java/com/facebook/presto/sql/analyzer/AnalysisContext.java

```diff
@@ -22,7 +23,7 @@
 public class AnalysisContext
 {
     private final AnalysisContext parent;
-    private final Map<String, Query> namedQueries = new HashMap<>();
+    private final Map<String, WithQuery> namedQueries = new HashMap<>();
```

Comment:
Not to have another Map<String, List<String>> (or a multimap) for column aliases, I changed it to WithQuery, because this used specifically for with queries, not views.